agent/hcp/client/mock_CloudConfig.go

```diff
@@ -2,14 +2,16 @@ package client
 
 import (
 	"crypto/tls"
-	"errors"
 	"net/url"
 
 	hcpcfg "github.com/hashicorp/hcp-sdk-go/config"
 	"github.com/hashicorp/hcp-sdk-go/profile"
+	"github.com/hashicorp/hcp-sdk-go/resource"
 	"golang.org/x/oauth2"
 )
 
+const testResourceID = "organization/ccbdd191-5dc3-4a73-9e05-6ac30ca67992/project/36019e0d-ed59-4df6-9990-05bb7fc793b6/hashicorp.consul.linked-cluster/prod-on-prem"
```

Comment:
Is this a real world org and project ID? I know it's just used in a unit test, but are we potentially "leaking" this data?

Reply:
Nope was a test one from the resource package, but good point! I modified it to use fake resource data to build a fake string